lib/Dialect/FIRRTL/Transforms/LowerSignatures.cpp

```diff
@@ -329,7 +330,8 @@ static LogicalResult lowerModuleSignature(FModuleLike module, Convention conv,
     // handled differently below.
     if (attr.getName() != "portNames" && attr.getName() != "portDirections" &&
         attr.getName() != "portTypes" && attr.getName() != "portAnnotations" &&
-        attr.getName() != "portSyms" && attr.getName() != "portLocations")
+        attr.getName() != "portSyms" && attr.getName() != "portLocations" &&
+        attr.getName() != "internalPaths")
```

Comment:
Pattern is already here, but since we have these attributes cached already maybe it'd be better (and less repeating of the strings) to instead compare against the `StringAttr` for quick pointer comparison instead?

Reply:
Agreed, but in a follow-up PR.